roofit/roofitcore/src/RooAbsArg.cxx

```diff
@@ -2229,6 +2229,13 @@ RooAbsArg* RooAbsArg::cloneTree(const char* newname) const
   RooAbsArg* head = clonedNodes.find(*this) ;
   assert(head);
 
+  // We better to release the ownership before removing the "head". Otherwise,
+  // "head" might also be deleted as the clonedNodes collection owns it.
+  // (Actually this does not happen because even an owning collection doesn't
+  // delete the element when removed by pointer lookup, but it's better not to
+  // rely on this unexpected fact).
+  clonedNodes.releaseOwnership();
```

Comment:
Maybe it's important to note that this line further down would have transferred the ownership anyway, mplicitely because of the `move`:
```C++
head->addOwnedComponents(std::move(clonedNodes)) ;
```

It's just better to release the ownership already before to make it explicit that `head` is not deleted.